force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -356,14 +449,20 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<Person> createMultiplePersons(List<String> names, Integer age) {
 
+        List<Person> persons = new List<Person>();
+
         // Loop through each name in the provided list of names
             // Create a new Person object for each name
-            
             // Add the new Person object to the list of Person objects
-        
+        for (String name : names) {
+            Person person = new Person(null, null);
+            person.setName(name);
+            person.setAge(age);
```

Comment:
Again, be sure to take advantage of the constructor. Adding null and then setting the name and age doesn't make much sense. Right? Remember, developers are lazy. No need for extra work! Ha.
```suggestion
            Person person = new Person(name, age);
```